src/styles/utils/variables.scss

```diff
@@ -0,0 +1,14 @@
+$base-size: 100px;
+$base-gap: 1px;
+$day-color: #eee;
+$hover-color: #ffbfcb;
+$day-names:
+(
+  'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
+)
```

Comment:
There is a syntax error in the declaration of the `$day-names` variable. In SCSS, lists are declared without parentheses. You should remove the parentheses around the list of day names and their corresponding numbers.